rllib/core/rl_module/encoder.py

```diff
@@ -13,6 +13,9 @@
 
 # TODO (Kourosh): Find a better / more straight fwd approach for sub-components
 
+ENCODER_OUT = "encoder_out"
```

Comment:
@kouroshHakha I don't know where yet, but we should standardize these somewhere at least for the stuff we build internally.

Reply:
For now let's go with this. We should define an Enum object and pull out all keys from SampleBatch class into that Enum object.